test/tst_loss.jl

```diff
@@ -172,21 +194,29 @@ end
 
 @testset "Test typestable supervised loss for type stability" begin
     for loss in [L1HingeLoss(), L2HingeLoss(), ModifiedHuberLoss(), PerceptronLoss(),
-                ZeroOneLoss(), LPDistLoss(1), LPDistLoss(2), LPDistLoss(3)]
+                LPDistLoss(1), LPDistLoss(2), LPDistLoss(3)]
         test_value_typestable(loss)
-        if typeof(loss) <: Union{MarginLoss, DistanceLoss}
-            test_value_floatforcing(2. * loss)
-        end
+        # TODO: add ZeroOneLoss after scaling works...
```

Comment:
yes, I remember. I will revisit the `ZeroOneLoss` sooner or later. This is perfectly fine for now